phpstan.neon

```diff
@@ -6,9 +6,10 @@ includes:
 
 parameters:
     autoload_files:
-        - %rootDir%/../../../vendor/yiisoft/yii2/Yii.php
+        - vendor/yiisoft/yii2/Yii.php
 
+    checkMissingIterableValueType: false
```

Comment:
🤷‍♀

Reply:
why?

Reply:
```
20     Method Proget\Tests\PHPStan\Yii2\Yii\MyActiveRecord::getMaster() return
         type has no value type specified in iterable type
         Proget\Tests\PHPStan\Yii2\Yii\MyActiveRecord.
         💡 Consider adding something like
         Proget\Tests\PHPStan\Yii2\Yii\MyActiveRecord<Foo> to the
         PHPDoc.
         You can turn off this check by setting
         checkMissingIterableValueType: false in your
         ./phpstan.neon.
```

No idea what to do with it, since we are overriding active query return type with a custom one, which is properly resolved and used for `->one()` method and is returning an object|null when `asArray` is not used on it.

Reply:
`MyActiveRecord<Foo>` make sens, try to use `@template` annotation :rocket: :sunglasses: